src/generators.rs

```diff
@@ -1831,6 +1831,16 @@ pub fn gen_stx_transfer(scale: u16) -> (Option<String>, String) {
     (None, body)
 }
 
+pub fn gen_stx_get_balance(scale: u16) -> (Option<String>, String) {
+    let mut body = String::new();
+
+    for _ in 0..scale {
+        body.push_str("(stx-get-balance 'S1G2081040G2081040G2081040G208105NK8PE5) ");
```

Comment:
Are you setting up a balance for this principal? Or do you just expect to get 0?

Reply:
yup! happens in `setup_chain_state`